app/code/Magento/Checkout/Block/Cart/Helper/ConfigProvider.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Checkout\Block\Cart\Helper;
+
+class ConfigProvider
```

Comment:
This class look more like a model than a block. Can you please move it to the Model namespace, also avoid `Helper` in class naming as it is too general, try to pick meaningful namespace and class name. Also, please add comment header for class and enable strict mode.